src/TrafficManager/TrafficManager/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+##Version 1.0.5
```

Comment:
You don't need to update version your self. Please just add changelog under `Upcoming Release`.